Cargo.toml

```diff
@@ -40,3 +40,9 @@ test = true
 bench = false
 name = "ci_test"
 test = true
+
+[[test]]
+bench = false
+harness = false
+name = "ci"
+test = true
```

Comment:
I see my mistake - I wasn't explicit enough :)  So... could you remove the unnecessary `test = true` lines please?